AppController/djinn.rb

```diff
@@ -5743,6 +5743,7 @@ def try_to_scale_down(app_name, delta_appservers)
           @last_decision[app_name] = Time.now.to_i
           Djinn.log_info("Removing an AppServer for #{app_name} #{location}.")
           num_to_remove -= 1
+          examine_scaling_down_instances(app_name, node_ip)
```

Comment:
Some of the appengine nodes from get_all_appengine_nodes might have other roles, though.